core-bundle/src/Resources/contao/dca/tl_search_index_queue.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+$GLOBALS['TL_DCA']['tl_search_index_queue'] = array
```

Comment:
We should probably make this a Doctrine entity (see https://github.com/contao/contao/blob/master/core-bundle/src/Entity/RememberMe.php)

Reply:
Done in 3bdd8dca12624b65697917717e64481e82760bd7. I've also properly named it because it's related to crawling and not the search index at all 😏